src/loaders/compute_tools/entrypoint.sh

```diff
@@ -7,6 +7,7 @@ echo "using $env_tool to manage environment"
 $env_tool run -n $CONDA_ENV \
   python $PY_SCRIPT \
   --load_ver $LOAD_VER \
+  --source_ver $SOURCE_VER \
```

Comment:
I guess we're going to have to rebuild & push the 3 tool images yeah?

Reply:
yup.